import "Robot/Public/Proto/serviceParams.proto";

package robot;
message RobotServiceInfo {
  required string bot_name = 1;
  repeated string nick_name = 2;
  optional ServiceParams params = 3;
}
message RobotServiceConfig {
  repeated RobotServiceInfo services = 1;
}
